packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -135,13 +146,16 @@ export class DebugSession implements Disposable {
     if (this.jsDebugSession) {
       await this.restart();
     }
+    if (this.options.useParentDebugSession && !this.parentDebugSession) {
+      await this.startParentDebugSession();
+    }
```

Comment:
Can this `if` condition ever be satisfied? We've just started the `parentDebugSession` in `restart`